src/compile.js

```diff
@@ -1,14 +1,6 @@
-/**
- * Send
- * Communicate an action to a store. This is how Microcosm determines
- * how transaction parameters should update state.
- */
-
 import { formatTag } from './tag'
-import { isFunction } from './type-checks'
-import { get } from './update'
 
-export default function send (key, store, state, type, payload) {
+function generate (key, store, type) {
```

Comment:
I find this function name confusing. Doesn't seem like you're really generating anything here.